### ISSUE TITLE
Can't define keys

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ionux/genplus-droid#1]
```
What steps will reproduce the problem?
1. Settings
2. Configure Key Input
3. Setting Key For Buttons

What is the expected output? What do you see instead?
Can't set up keys with my Eee Pad Transformer

What version of the product are you using? On what operating system?
ASUS Eee Pad Transforoidrmer - Android 3.2
GENPlusD 1.5

Please provide any additional information below.
The Cursor Keys on the Transformer keyboard works fine, but if I try to define 
the keys for GENPlus every time they will be define as 0, or -1.
I navigate to Configure Key Input and select the A Button, but can't press a 
new key on the keyboard.

A fix would be fine for playing games on an Android device incl. a keyboard.
```

Original issue reported on code.google.com by `claus.mo...@gmail.com` on 8 Aug 2011 at 9:09

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in version 1.8.
```

Original comment by `shinhals...@gmail.com` on 28 Sep 2011 at 5:01
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
...
```

Original comment by `shinhals...@gmail.com` on 28 Sep 2011 at 5:05
